backends/ebpf/ebpfTable.h

```diff
@@ -104,6 +104,7 @@ class EBPFTable : public EBPFTableBase {
     virtual void emitValueType(CodeBuilder* builder);
     virtual void emitValueActionIDNames(CodeBuilder* builder);
     virtual void emitValueStructStructure(CodeBuilder* builder);
+    virtual void emitDirectTypes(CodeBuilder* builder) { (void) builder; }
```

Comment:
Why do we put `emitDirectTypes` in a classic EBPF table? I'm not sure if there is a plan for `ebpf_model` to implement DirectCounters..

Reply:
It is called inside `emitValueType`, so moving to PSA table it would require rewrite also that function.